### PR TITLE
chore(ci): update google-github-actions to v2

### PR DESCRIPTION
**Pull request by balzdur**
In this PR :
- inject sentry env vars (since they are not created anymore by the previous dedicated action)
- update some actions to resolve warnings from the CI (due to Node version updation in Github Actions)

### CODE REVIEW

File: .github/workflows/build_and_deploy.yaml [hunk 1]
```diff
@@ -55,19 +55,19 @@ jobs:
...
-        uses: 'google-github-actions/auth@v1'
+        uses: 'google-github-actions/auth@v2'
         with:
           credentials_json: '${{ secrets.GCP_SA_KEY }}'
 
       - name: 'Set up Cloud SDK'
-        uses: 'google-github-actions/setup-gcloud@v1'
+        uses: 'google-github-actions/setup-gcloud@v2'
         with:
           install_components: beta
```
Comment (balzdur) [inline .github/workflows/build_and_deploy.yaml:72]: Not sure we still need those but this is out of scope

Comment (Pascal-Delange) [reply .github/workflows/build_and_deploy.yaml:72]: yep I think we do, it's what's used in the next step

**Review by Pascal-Delange — APPROVED** [review]
